lib/dor/text_extraction/abbyy/ticket.rb

```diff
@@ -30,23 +34,27 @@ def pdf_files?
           File.extname(filepaths.first).strip.downcase[1..] == 'pdf'
         end
 
+        def input_file_path
+          File.join(Settings.sdr.abbyy_ticket_path, bare_druid)
+        end
+
         def output_file_path
```

Comment:
Is this method used externally by someone else?  It doesn't appear to  used in this class so could be removed if not.  If it is used elsewhere, it could then be used below in `#output_file_path` to avoid code duplication.

Reply:
I was going to use it in the fetch-files workflow step. but in hindsight it's more trouble than it's worth